### PR TITLE
Supply chain update solarwinds

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Adding solarwinds attack here.

### CODE REVIEW

**Comment by TheFoxAtWork** [comment]
@SantiagoTorres

File: supply-chain-security/compromises/2020/solarwinds.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* Middle East
+* Asia
+
+This compromise permitted data exfiltration and lateral movement through
+Solarwinds customer environment via the Orion monitoring tool.
+
+## Type of compromise
+
+While the specific details of how initial access occurred, initially it is
+understood that multiple compromises may have occurred:
```
Comment (joshuagl) [inline supply-chain-security/compromises/2020/solarwinds.md:45]: This is a bit jumbled, perhaps
```suggestion
While the specific details of how initial access occurred are unknown, it is
understood that multiple compromises may have occurred:
```

**Review by joshuagl — COMMENTED** [review]
Thanks for the swift writeup. This is a good summary.

**Comment by ultrasaurus** [comment]
Great write up!  Agree it is good to include.  One question (I haven’t read all linked articles, so apologies if this is obvious to others) — is it know that  “Attackers signed libraries using Solarwinds own certificates” or could it be upstream issue?   If known, pls reference source as with other bullet points.

thanks!

**Comment by TheFoxAtWork** [comment]
> Great write up! Agree it is good to include. One question (I haven’t read all linked articles, so apologies if this is obvious to others) — is it know that “Attackers signed libraries using Solarwinds own certificates” or could it be upstream issue? If known, pls reference source as with other bullet points.
> 
> thanks!
@ultrasaurus Its still unknown what precisely was the cause.  They know the certificates were compromised as a result of either the dev tooling or publishing infrastructure being compromised.  It could go the path of [CCleaner](https://www.wired.com/story/inside-the-unnerving-supply-chain-attack-that-corrupted-ccleaner/) but its too early to tell currently.

**Comment by SantiagoTorres** [comment]
This LGTM!

Thank you for the nice writeup @TheFoxAtWork !

**Review by ultrasaurus — APPROVED** [review]
Approve!  

For future:  @TheFoxAtWork and @SantiagoTorres should be OWNERS of this -- I think I had volunteered to add something to governance/roles allowing for there to be project leads of on-going projects with OWNER role of a directory -- I'll open an issue

**Pull request merged**